python/tvm/auto_scheduler/relay_integration.py

```diff
@@ -315,11 +315,10 @@ def auto_schedule_topi(outs):
             new_key = json.dumps((new_dag.hash_key(),))
             if new_key != key:
                 dispatch_ctx.update(target, new_key, state)
-        return te.create_schedule([x.op for x in outs])
     else:
         raise ValueError("Invalid tracing mode: " + env.tracing_mode)
```

Comment:
If program stop at this error exception, will the None return value take effect?

Reply:
No, but if it reaches to this point then it must be something wrong with the code, because we should not expect an invalid tracing mode.